modules/specifying-oc-log-levels.adoc

```diff
@@ -8,22 +8,37 @@
 
 You can investigate OpenShift CLI (`oc`) issues by increasing the command's log level.
 
+The {product-title} user's current session token is typically included in logged `curl` requests where required. You can also obtain the current user's session token manually, for use when testing aspects of an `oc` command's underlying process step by step.
+
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Specify the `oc` log level when running an `oc` command:
+* Specify the `oc` log level when running an `oc` command:
 +
 [source,terminal]
 ----
-$ oc <options> --loglevel <log_level>
+$ oc <command> --loglevel <log_level>
 ----
++
+where:
++
+--
+<command>:: Specifies the command you are running.
+<log_level>:: Specifies the log level to apply to the command.
+--
 
-. The {product-title} user's current session token is typically included in logged `curl` requests where required. You can also obtain the current user's session token manually, for use when testing aspects of an `oc` command's underlying process step by step:
+* To obtain the current user's session token, run the following command:
 +
 [source,terminal]
 ----
 $ oc whoami -t
 ----
++
+.Example output
+[source,terminal]
```

Comment:
Per the latest [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc) to avoid unhelpful highlighting:
```suggestion
[source,text]
```